pkg/test-infra/tidb/recommend.go

```diff
@@ -107,7 +110,7 @@ func RecommendedTiDBCluster(ns, name string) *Recommendation {
 					},
 				},
 				TiKV: v1alpha1.TiKVSpec{
-					Replicas:             3,
+					Replicas:             *TiKVReplicas,
```

Comment:
polish?

Reply:
Here maybe better to use the same way like we do in `MaxFailoverCount`. Since now tikv replicas is configurable, maybe we need to make `MaxFailoverCount` configurable as well
``` Go
pointer.Int32Ptr(3)
```